File: src/DutyFree.Entity/ValueType/CartItem.cs
namespace DutyFree.Entity.ValueType
{
    public class CartItem
    {
        public int Quantity { get; set; }
        public bool IsSelected { get; set; }

        public CartItem(int quantity, bool isSelected)
        {
            this.Quantity = quantity;
            this.IsSelected = isSelected;
        }
    }
}

File: bomeiboyou/src/entity/cart-item.ts
export class CartItem {
    public constructor(public productId: string, public quantity: number, public isSelected: boolean,
        public logo: string, public name: string, public marketPrice: number, public price: number,
        public commentId: string, public orderInfoId: string, public referencePrice: number, public minPrice: number,
        public maxPrice: number) { }
}